src/LoginScreen.js

```diff
@@ -1,9 +1,61 @@
-import React from 'react';
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {loginWithUserData} from './helperFunctions/login';
+import {updateJWT} from './actions';
+import './stylesheets/login.css';
 
-let LoginScreen = () => 
-    <div>
-        <p>Login Screen</p>
-    </div>
 
 
+let mapStateToProps = (state) => {
```

Comment:
I saw in your initial state you have "is logged in".  Maybe instead of mapping the JWT, we should just map that and always get the token from local storage when we need it?  The advantage to local storage is that it will survive the refresh.

Reply:
This part of state helps the header to know whether to show the login /create account option (if userloggedin = false) or the profile/queue option

Reply:
i don't have an argument for storing jwt in state vs localstorage vs both except that we will need it for all of our private routes and it might be easier?

Reply:
that's fine.  I didn't understand that it was intended to hold user information as well.  Still, I think the token should always be pulled from local storage during fetches.